package.json

```diff
@@ -97,5 +100,8 @@
     "react-component",
     "select",
     "ui"
-  ]
+  ],
+  "lint-staged": {
+    "*.js": ["eslint", "git add"]
```

Comment:
What is the point of `git add` here? unless `eslint` is being run with the `--fix` flag (which it's not and I'm happy to leave it that way) this wouldn't be necessary; and it has the side-effect of preventing you from committing only staged chunks of a file without making it an obvious thing...

Surely just `eslint` on its own would be sufficient (and safer!) here?

Reply:
I knew that I didn't want to have it fix linter errors automatically but I didn't really understand what git add was doing. Reading further down the readme of lint-staged helped with that 😛. I totally agree with what you're saying here so making the change.